.travis.yml

```diff
@@ -1,12 +1,23 @@
 language: go
 services:
-    - redis-server
+- redis-server
 go:
-  - 1.4
-install: ./scripts/build-setup.sh
-script: 
-    - ./scripts/build-build.sh
-    - ./scripts/test.sh
+- 1.4
+install: sleep 0 # Travis does go get -t -v- ./... which we don't want.
+script:
+- ./scripts/build.sh
```

Comment:
Also one other thought, if we are building on travis, we need to make sure that build will be compatible with our version of linux. I was unable to build on ubuntu and run on centos in the past.